README.md

```diff
@@ -41,3 +41,19 @@ W celu automatycznej kompilacji plików .coffee należy wymagane jest skonfiguro
 - wybrać z menu File/Settings/File Watchers
 - kliknąć '+' i wybrać 'CoffeeScript'
 - pole 'Program' powinno zawierać ścieżkę do zainstalowanego wcześniej transpilera. Jeżeli nie - należy podać odpowiednią ścieżkę.
+
+Redis
+======
+
+W celu umożliwienia współdzielenia sesji na różnych nodach klastra, serwer korzysta 
+z bazy Redis. Informacje połączenia pobierane są ze zmiennej środowiskowej
+exportowanej przez heroku: REDISTOGO_URL. 
+
+Na potrzeby developmentu i szybkiego uruchamiania aplikacji na jednym węźle,
+można skorzystać z wbudowanej bazy Redis, która aktywuje się po włączeniu profilu
+springowego "dev". Można to zrobić np. poprzez dodanie:
+
+```
+   Intellij / Run / Edit Configurations... / Tomcat Server / <config_name> / Startup/Connection / Environment Variables / Add
+   spring.profiles.active=dev
```

Comment:
still getting:
 org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'enableRedisKeyspaceNotificationsInitializer' defined in class path resource [org/springframework/session/data/redis/config/annotation/web/http/RedisHttpSessionConfiguration.class]: Invocation of init method failed; nested exception is org.springframework.data.redis.RedisConnectionFailureException: Cannot get Jedis connection; nested exception is redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool